src/main/webapp/i18n/de/userSettings.json

```diff
@@ -0,0 +1,46 @@
+{
+    "artemisApp": {
+        "userSettings": {
+            "settings": "Benachrichtigungseinstellungen",
+            "saveChanges": "Änderungen speichern",
+            "saveSettingsSuccessAlert": "Die Einstellungen wurden erfolgreich gespeichert.",
+            "userSettings": "Benutzereinstellungen",
+            "accountInformation": "Account Informationen",
+            "notificationSettings": "Benachrichtigungen",
+            "accountInformationPage": {
+                "fullName": "Vollständiger Name",
+                "login": "Login",
+                "email": "Email",
+                "joinedArtemis": "Artemis beigetreten am"
+            },
+            "categories": {
+                "NOTIFICATION_SETTINGS": "Benachrichtigungseinstellungen"
+            },
+            "optionGroupNames": {
+                "Exercise Notifications": "Aufgabenbenachrichtigungen",
+                "Lecture Notifications": "Vorlesungsbenachrichtigungen",
+                "Instructor Exclusive Notifications": "Exklusive Benachrichtigungen für Lehrende"
+            },
+            "optionNames": {
+                "Exercise created or started": "(Neue) Aufgabe erstellt oder gestartet",
+                "Exercise open for Practice": "Aufgabe zum Üben freigegeben",
+                "New Post for Exercises": "Neue Beiträge für Aufgaben",
+                "New Reply for Exercises": "Neue Antworten für Aufgaben",
+                "Attachment Changes": "Änderungen von Anhängen",
+                "New Post for Lecture": "Neue Beiträge für Vorlesungen",
+                "New Reply for Lecture": "Neue Antworten für Vorlesungen",
+                "Course and Exam Archiving Started": "Kurs und Klausur Archivierungsstart"
```

Comment:
why not using actual translation keys such as "ExerciseCreatedOrStarted", "ExerciseOpenForPractice"...

Reply:
Please have a look at my response above